benchmark/benchmark_hydrostatic_model.jl

```diff
@@ -103,10 +103,8 @@ grid_types = [
 free_surface_types = [
     :MatrixImplicitFreeSurface,
     :ExplicitFreeSurface,
-    # ImplicitFreeSurface doesn't yet work on MultiRegionGrids like the ConformalCubedSphereGrid:
-    :FFTImplicitFreeSurface, 
     :ImplicitFreeSurface,
-    # :ImplicitFreeSurface
+    :NoneImplicitFreeSurface
```

Comment:
Can we use `nothing` rather than adding a new type?

Reply:
you mean `nothingImplicitFreeSurface` here? Or switch `preconditioner_method = :None` to `preconditioner_method = nothing`?